voltixApp/VoltixApp.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -1,41 +0,0 @@
-{
-  "pins" : [
-    {
```

Comment:
I don't think you can remove this one , otherwise build will fail

Reply:
It is generated in runtime